### PR TITLE
[MINOR] Fix deployment issues

### PR DESCRIPTION
**Pull request by jongyoul**
### What is this PR for?
Fixing some issues that happened when releasing.

### What type of PR is it?
Hot Fix

### Todos
* [x] - Remove some plugins from the shell interpreter instead of removing them from the distribution as some interpreters depend on it
* [x] - Upgrade python version for docker image to 3.9

### What is the Jira issue?
N/A

### How should this be tested?
Distribution and building docker image should work fine.

### Questions:
* Does the license files need to update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: shell/pom.xml [hunk 1]
```diff
@@ -125,9 +125,9 @@
       </plugin>
       <plugin>
         <artifactId>maven-resources-plugin</artifactId>
-      </plugin>
-      <plugin>
-        <artifactId>maven-shade-plugin</artifactId>
+        <configuration>
```
Comment (pan3793) [inline shell/pom.xml:128]: what's the effect of this change?

Comment (pan3793) [reply shell/pom.xml:128]: It seems the whole `sh` dir disappears in the `interpreter` folder, not only "configurations", is it expected?

Comment (jongyoul) [reply shell/pom.xml:128]: Yes, correct. Shell interpreter was and will be removed from the official binary because it lead a lot of security issues.

Comment (pan3793) [reply shell/pom.xml:128]: LGTM if it's the expected behavior. Test may be need to adjusted to adapt this change.

**Comment by jongyoul** [comment]
There are a lot of test failures. Let me fix it one by one soon

**Pull request closed without merging**